src/plonetheme/eeq/browser/overrides/jazkarta.tesserae.templates.collection.pt

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xml:lang="en"
+      lang="en"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="plone">
+  <body tal:define="collection nocall:view/content;
+                    url collection/absolute_url|nothing;
+                    title collection/Title|nothing;
+                    description collection/Description|nothing;
+                    results view/results;">
+    <div class="cards collection-tile-cards" tal:condition="results">
```

Comment:
Looks like this customization just adds the `collection-tile-cards` class. Is that necessary? Can't we just style `.cards` or `.mosaic-jazkarta\.tesserae\.collection_summary-tile .mosaic-tile-content .cards` and skip this override? Is there some other customization here that I'm missing?

Reply:
Right. I thought I needed to customize it and forgot to remove it when I should have seen it was not necessary. Fixed. Thanks!